internal/photoprism/convert_sidecar_json.go

```diff
@@ -58,7 +58,39 @@ func (c *Convert) ToJson(f *MediaFile, force bool) (jsonName string, err error)
 
 	// Check if file exists.
 	if !fs.FileExists(jsonName) {
-		return "", fmt.Errorf("exiftool: failed creating %s", filepath.Base(jsonName))
+		return "", fmt.Errorf(
+			"exiftool: failed creating %s", filepath.Base(jsonName),
+		)
+	}
+
+	// attempt to extract the video file if it exists
+	var extractCmd *exec.Cmd
+	path := filepath.Join(
+		Config().SidecarPath(), f.RootRelPath(), "%f"+fs.ExtMP4,
+	)
+
+	if f.IsJpeg() {
```

Comment:
Is this performed for all JPEGs? If so, it could slow down indexing and should only be executed if the file is likely to contain an embedded video, e.g. based on the previously extracted metadata.